<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_neuesGemuese">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Erzeuge neues Gemuese</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txt_neuesGemuese">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_alleGemuese">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>131</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_leseGemueseListe">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>241</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Lade Gemuese</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txt_saeen">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_saeen">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wann säen?</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_vorzucht">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vorzucht:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txt_vorzucht">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_speichereGemueseJson">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>320</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Speichere Neue Einträge</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pflanzabstnde">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pflanzabstände:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pflanzabstaende_inReihe">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>in einer Reihe:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pflanzabstaende_Reihen">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>zwischen Reihen:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_nachbarn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gute Nachbarn: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_kommentare">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Kommentare und Besonderheiten:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txt_nachbarn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txt_pflanzabstaende_inReihe">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txt_pflanzabstaende_Reihen">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txt_kommentare">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_erfahrungsbereich">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Erfahrungsbereich:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_beachtenBeiNeuerPflanzung">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Zu Beachten bei neuer Pflanzung: </string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txt_beachtenBeiNeuerPflanzung">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>260</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
